src/main/resources/application.yml

```diff
@@ -12,7 +12,11 @@ spring:
     date-format: "yyyy-MM-dd HH:mm:ss"
     serialization:
       WRITE_DATES_AS_TIMESTAMPS: false
-
+  security:
+    oauth2:
+      resourceserver:
+        jwt:
+          jwk-set-uri: ${oauth.endpoint.url}/.well-known/jwks.json
```

Comment:
Actually that's much better than what we have done before - which was setting this url for each env